pyhomematic/devicetypes/thermostats.py

```diff
@@ -151,6 +177,12 @@ def __init__(self, device_description, proxy, resolveparamsets=False):
         self.SENSORNODE.update({"TEMPERATURE": [1],
                                 "HUMIDITY": [1]})
         self.WRITENODE.update({"SETPOINT": [2]})
+        # Remove invalid parameters
+        self.ATTRIBUTENODE.pop("BATTERY_STATE", None)
+        self.ATTRIBUTENODE.pop("CONTROL_MODE", None)
+        self.SENSORNODE.pop("ACTUAL_TEMPERATURE", None)
+        self.SENSORNODE.pop("ACTUAL_HUMIDITY", None)
+        self.WRITENODE.pop("SET_TEMPERATURE", None)
```

Comment:
I don't see where these parameters could be set for ThermostatWall2 so it doesn't make sense to remove them. Leftover from remote testing?